jenkins.sh

```diff
@@ -8,13 +8,12 @@ set -e
 function install_required_packages {
   echo "print(.libPaths())" >> installPackages.R
   echo "try(remove.packages('synapser'), silent=T)" > installPackages.R
-  echo "try(remove.packages('PythonEmbedInR'), silent=T)" >> installPackages.R
-  echo "install.packages(c('pack', 'R6', 'testthat', 'knitr', 'rmarkdown', 'PythonEmbedInR'), " >> installPackages.R
+  echo "install.packages(c('pack', 'R6', 'testthat', 'knitr', 'rmarkdown', 'reticulate'), " >> installPackages.R
 
   # we use no-lock throughout. there should never be two executors running on the same
   # label at the same time, but if we use locks an aborted jenkins build can leave stale
   # locks that will break subsequent builds until cleaned up.
-  echo "repos=c('http://cran.fhcrc.org', '${RAN}'), INSTALL_opts='--no-lock')" >> installPackages.R
+  echo "repos=c('https://cran.r-project.org', '${RAN}'), INSTALL_opts='--no-lock')" >> installPackages.R
```

Comment:
Are we allowed to deploy on cran?

Reply:
We are free to submit, and as long as we meet the guidelines, I don't see why we shouldn't be. The project currently has 5 warnings due to documentation issues that we're working through that may prevent acceptance in the immediate term, so it may be preferable to set this to Sage's cran mirror while we work on that. We were looking to remove fhcrc.org as it is intermittently unreliable, and may have jumped the gun in this one place.

Reply:
Here are some useful links on CRAN submission
https://cran.r-project.org/web/packages/policies.html
https://cran.r-project.org/web/packages/submission_checklist.html

Reply:
Ya... is it feasible to contribute this to cran in the tight timeframe we have?  We can set it to Sage's CRAN mirror for now.